### PR TITLE
Add instructions for deploying migration assistant.

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 1]
```diff
@@ -7,66 +7,35 @@ parent: Deploying migration assistant
...
-3. **Live Capture Migration with Capture and Replay (C&R)**
+This page outlines the configuration options for three key migrations scenarios:
 
-Each of these migrations may depend on either a snapshot or a capture proxy. The CDK context blocks below are shown as separate context blocks for each migration type for simplicity. If performing multiple migration types, combine these options, as the actual execution of each migration is controlled from the Migration Console.
+1. **Metadata migration**
+2. **Backfill migration with `Reindex-from-Snapshot` (RFS)**
+3. **Live capture migration with Capture and Replay (C&R)**
 
-It also has a section describing how to specify the auth details for the source and target cluster (no auth, basic auth with a username and password, or sigv4 auth).
+Each of these migrations depend on either a snapshot or a capture proxy. They are shown as AWS Cloud Deployment Kits (CDKs). The CDKs on this page are shown as separate blocks for each migration type. If you are performing a migration that suits multiple scenarios, these options can be combined in the migration console.
```
Comment (peternied) [inline _migrations/deploying-migration-assistant/configuration-options.md:16]:
```suggestion
Each of these migrations depend on either a snapshot or a capture proxy. There are example cdk.context.json configurations that are used by AWS Cloud Deployment Kits (CDKs) to deploy and configure the Migration Assistant. The example cdk.context.json configurations on this page are shown as separate blocks for each migration type. If you are performing a migration that suits multiple scenarios, these options can be combined.
```

Yea... this is a confusing workflow.  So there is a file called `cdk.context.json` that is used to provided parameters to the CDK tool for deploying Migration Assistant.  Changing the parameters following the recommendations below changes what is deployed and how it is configured.  So I'd call these examples parameter templates or deployment context templates.  After a user configures the cdk.context.json, then they deploy with CDK and then they can start using the MA via the migration console.

Also tweaked that final statement, the migration console is the 'control panel' for all these different migration types and sadly it cannot update the deployed features.

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 2]
```diff
@@ -93,22 +62,22 @@ There are currently no CDK options specific to Metadata migrations, which are pe
...
-| `reindexFromSnapshotServiceEnabled` | `true`                                                                | Enables deploying and configuring the RFS ECS service.                                                                                                                                                      |
-| `reindexFromSnapshotExtraArgs`      | `"--target-aws-region us-east-1 --target-aws-service-signing-name es"` | Extra arguments for the Document Migration command, with space separation. See the [RFS Extra Arguments](https://github.com/opensearch-project/opensearch-migrations/blob/main/DocumentsFromSnapshotMigration/README.md#arguments) for more details. You can pass `--no-insecure` to remove the `--insecure` flag. |
+The RFS CDK uses the following options. To view all available arguments for `reindexFromSnapshotExtraArgs`, see [Snapshot migrations README](https://github.com/opensearch-project/opensearch-migrations/blob/main/DocumentsFromSnapshotMigration/README.md#arguments). At a minimum, no extra arguments may be needed.
```
Comment (peternied) [inline _migrations/deploying-migration-assistant/configuration-options.md:68]:
```suggestion
RFS configuration uses the following options. To view all available arguments for `reindexFromSnapshotExtraArgs`, see [Snapshot migrations README](https://github.com/opensearch-project/opensearch-migrations/blob/main/DocumentsFromSnapshotMigration/README.md#arguments). At a minimum, no extra arguments may be needed.
```

What do you think, this work better?  Open to other approaches.

**Review by peternied — APPROVED** [review]

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 3]
```diff
@@ -7,66 +7,35 @@ parent: Deploying migration assistant
...
 
-> [!TIP]
-For a complete list of configuration options, please refer to the [opensearch-migrations options.md](https://github.com/opensearch-project/opensearch-migrations/blob/main/deployment/cdk/opensearch-service-migration/options.md) but please open an issue for consultation if changing an option that is not listed on this page.
 
-Options for the source cluster endpoint, target cluster endpoint, and existing VPC should be configured for the Migration tools to function effectively.
-
-
-## Metadata Migration Options
+For a complete list of configuration options, please refer to the [opensearch-migrations-options.md](https://github.com/opensearch-project/opensearch-migrations/blob/main/deployment/cdk/opensearch-service-migration/options.md). If there is a configuration option you would like that is not found on this page, open an issue in OpenSearch Migrations repository.
```
Comment (Naarcha-AWS) [inline _migrations/deploying-migration-assistant/configuration-options.md:19]:
```suggestion
For a complete list of configuration options, see [opensearch-migrations-options.md](https://github.com/opensearch-project/opensearch-migrations/blob/main/deployment/cdk/opensearch-service-migration/options.md). If there is a configuration option you would like that is not found on this page, open an issue in OpenSearch Migrations repository.
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 4]
```diff
@@ -137,28 +106,30 @@ Performing a Reindex-from-Snapshot backfill migration requires an existing snaps
...
-| `captureProxyExtraArgs`         | `"--suppressCaptureForHeaderMatch user-agent .*elastic-java/7.17.0.*"`                 | Extra arguments for the Capture Proxy command, including options specified by the [Capture Proxy](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java).                         |
-| `trafficReplayerServiceEnabled` | `true`                                                                                 | Enables the Traffic Replayer service deployment via a new CloudFormation stack.                                                                                                                                                                   |
+| Name  | Example  | Description   |
+| :--- | :--- | :--- |
+| `captureProxyServiceEnabled`    | `true`  | Enables the Capture Proxy service deployment via a new CloudFormation stack.  |
```
Comment (Naarcha-AWS) [inline _migrations/deploying-migration-assistant/configuration-options.md:117]:
```suggestion
| `captureProxyServiceEnabled`    | `true`  | Enables the Capture Proxy service deployment using a CloudFormation stack.  |
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 5]
```diff
@@ -137,28 +106,30 @@ Performing a Reindex-from-Snapshot backfill migration requires an existing snaps
...
-## Cluster Authentication Options
+For arguments available in `captureProxyExtraArgs`, refer to the `@Parameter` fields [here](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java). For `trafficReplayerExtraArgs`, refer to the `@Parameter` fields [here](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java). At a minimum, no extra arguments may be needed.
+
 
-Both the source and target cluster can use no authentication (e.g. limited to the VPC), basic authentication with a username and password, or SigV4 scoped to a user or role.
 
-Examples of each of these are below.
+## Cluster authentication pptions
+
+Both the source and target cluster can use no authentication, such as authentication limited to VPC, basic authentication with a username and password, or SigV4 scoped to a user or role.
```
Comment (Naarcha-AWS) [inline _migrations/deploying-migration-assistant/configuration-options.md:129]:
```suggestion
Both the source and target cluster can use no authentication, such as authentication limited to VPC, basic authentication with a username and password, or AWS Signature Version 4 scoped to a user or role.
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 6]
```diff
@@ -93,22 +62,21 @@ There are currently no CDK options specific to Metadata migrations, which are pe
...
-| `reindexFromSnapshotServiceEnabled` | `true`                                                                | Enables deploying and configuring the RFS ECS service.                                                                                                                                                      |
-| `reindexFromSnapshotExtraArgs`      | `"--target-aws-region us-east-1 --target-aws-service-signing-name es"` | Extra arguments for the Document Migration command, with space separation. See the [RFS Extra Arguments](https://github.com/opensearch-project/opensearch-migrations/blob/main/DocumentsFromSnapshotMigration/README.md#arguments) for more details. You can pass `--no-insecure` to remove the `--insecure` flag. |
+| Name  | Example | Description |
+| :--- | :--- | :--- |
+| `reindexFromSnapshotServiceEnabled` | `true` | Enables deploying and configuring the RFS ECS service. |
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/configuration-options.md:72]:
```suggestion
| `reindexFromSnapshotServiceEnabled` | `true` | Enables deployment and configuration of the RFS ECS service. |
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 7]
```diff
@@ -93,22 +62,21 @@ There are currently no CDK options specific to Metadata migrations, which are pe
...
+| `reindexFromSnapshotExtraArgs` | `"--target-aws-region us-east-1 --target-aws-service-signing-name es"` | Extra arguments for the Document Migration command, with space separation. See the [RFS Extra Arguments](https://github.com/opensearch-project/opensearch-migrations/blob/main/DocumentsFromSnapshotMigration/README.md#arguments) for more details. You can pass `--no-insecure` to remove the `--insecure` flag. |
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/configuration-options.md:73]:
```suggestion
| `reindexFromSnapshotExtraArgs` | `"--target-aws-region us-east-1 --target-aws-service-signing-name es"` | Extra arguments for the Document Migration command, with space separation. See [RFS Extra Arguments](https://github.com/opensearch-project/opensearch-migrations/blob/main/DocumentsFromSnapshotMigration/README.md#arguments) for more information. You can pass `--no-insecure` to remove the `--insecure` flag. |
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 8]
```diff
@@ -93,22 +62,21 @@ There are currently no CDK options specific to Metadata migrations, which are pe
...
 
-</details>
+To view all available arguments for `reindexFromSnapshotExtraArgs`, see [Snapshot migrations README](https://github.com/opensearch-project/opensearch-migrations/blob/main/DocumentsFromSnapshotMigration/README.md#arguments). At a minimum, no extra arguments may be needed.
 
-## Live Capture Migration with Capture and Replay (C&R) Options
+## Live capture migration with C&R 
 
-### Sample Live Capture Migration CDK Options
+The following sample CDK performs a live capture migration with C&R
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/configuration-options.md:79]:
```suggestion
The following sample CDK performs a live capture migration with C&R:
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 9]
```diff
@@ -137,28 +105,26 @@ Performing a Reindex-from-Snapshot backfill migration requires an existing snaps
...
-| `captureProxyExtraArgs`         | `"--suppressCaptureForHeaderMatch user-agent .*elastic-java/7.17.0.*"`                 | Extra arguments for the Capture Proxy command, including options specified by the [Capture Proxy](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java).                         |
-| `trafficReplayerServiceEnabled` | `true`                                                                                 | Enables the Traffic Replayer service deployment via a new CloudFormation stack.                                                                                                                                                                   |
+| Name  | Example  | Description   |
+| :--- | :--- | :--- |
+| `captureProxyServiceEnabled`    | `true`  | Enables the Capture Proxy service deployment using a CloudFormation stack.  |
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/configuration-options.md:113]:
```suggestion
| `captureProxyServiceEnabled`    | `true`  | Enables the Capture Proxy service deployment using an AWS CloudFormation stack.  |
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 10]
```diff
@@ -137,28 +105,26 @@ Performing a Reindex-from-Snapshot backfill migration requires an existing snaps
...
+| `captureProxyExtraArgs`  | `"--suppressCaptureForHeaderMatch user-agent .*elastic-java/7.17.0.*"`  | Extra arguments for the Capture Proxy command, including options specified by the [Capture Proxy](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java).  |
+| `trafficReplayerServiceEnabled` | `true`  | Enables the Traffic Replayer service deployment using a  CloudFormation stack.  |
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/configuration-options.md:115]:
```suggestion
| `trafficReplayerServiceEnabled` | `true`  | Enables the Traffic Replayer service deployment using a CloudFormation stack.  |
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 11]
```diff
@@ -137,28 +105,26 @@ Performing a Reindex-from-Snapshot backfill migration requires an existing snaps
...
 | `trafficReplayerExtraArgs`      | `"--sigv4-auth-header-service-region es,us-east-1 --speedup-factor 5"`                 | Extra arguments for the Traffic Replayer command, including options for auth headers and other parameters specified by the [Traffic Replayer](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java). |
 
-</details>
 
-## Cluster Authentication Options
+For arguments available in `captureProxyExtraArgs`, see the `@Parameter` fields [in the `CaptureProxy.java`](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java). For `trafficReplayerExtraArgs`, see the `@Parameter` fields [in TrafficReplayer.java](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java).
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/configuration-options.md:119]:
```suggestion
For arguments available in `captureProxyExtraArgs`, see the `@Parameter` fields in [`CaptureProxy.java`](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java). For `trafficReplayerExtraArgs`, see the `@Parameter` fields in [TrafficReplayer.java](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java).
```

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 12]
```diff
@@ -137,28 +105,26 @@ Performing a Reindex-from-Snapshot backfill migration requires an existing snaps
...
+
+
+## Cluster authentication options
 
-Both the source and target cluster can use no authentication (e.g. limited to the VPC), basic authentication with a username and password, or SigV4 scoped to a user or role.
+Both the source and target cluster can use no authentication, authentication limited to VPX, basic authentication with a username and password, or AWS Signature Version 4 scoped to a user or role.
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/configuration-options.md:124]: "limited to VPX" => "limited to a VPC"?

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 13]
```diff
@@ -180,7 +147,8 @@ Basic auth:
     }
 ```
 
-SigV4 auth:
+### SigV4 authentication
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/configuration-options.md:150]:
```suggestion
### Signature Version 4 authentication
```

File: _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md [hunk 14]
```diff
@@ -7,25 +7,27 @@ parent: Deploying migration assistant
 
 # IAM and security groups for existing clusters
 
-This page outlines scenarios for using the migration tools with existing clusters, including any necessary configuration changes to ensure proper communication between them.
+This page outlines security scenarios for using the migration tools with existing clusters, including any necessary configuration changes to ensure proper communication between them.
 
-## Importing an OpenSearch Service or OpenSearch Serverless Target Cluster
+## Importing an OpenSearch Service or OpenSearch Serverless target cluster
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md:12]:
```suggestion
## Importing an Amazon OpenSearch Service or Amazon OpenSearch Serverless target cluster
```

File: _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md [hunk 15]
```diff
@@ -7,25 +7,27 @@ parent: Deploying migration assistant
...
 
-### OpenSearch Service
+Use the following scenarios for OpenSearch Service or Serverless target clusters:
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md:14]:
```suggestion
Use the following scenarios for Amazon OpenSearch Service or Amazon OpenSearch Serverless target clusters.
```

File: _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md [hunk 16]
```diff
@@ -7,25 +7,27 @@ parent: Deploying migration assistant
...
+
+### OpenSearch service
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md:16]:
```suggestion
### OpenSearch Service
```

File: _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md [hunk 17]
```diff
@@ -7,25 +7,27 @@ parent: Deploying migration assistant
...
 
 For an OpenSearch Domain, two main configurations are typically required to ensure proper functioning of the migration solution:
 
-1. **Security Group Configuration**:  
-   The Domain should have a security group that allows communication from the applicable Migration services (Traffic Replayer, Migration Console, Reindex-from-Snapshot). The CDK will automatically create an `osClusterAccessSG` security group, which is applied to the Migration services. The user should then add this security group to their existing Domain to allow access.
+1. **Security Group Configuration**
+
+   The Domain should have a security group that allows communication from the applicable Migration services (Traffic Replayer, Migration Console, `Reindex-from-Snapshot`). The CDK automatically creates an `osClusterAccessSG` security group, which is applied to the migration services. The user should then add this security group to their existing Domain to allow access.
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md:22]:
```suggestion
   The domain should have a security group that allows communication from the applicable migration services (Traffic Replayer, Migration Console, `Reindex-from-Snapshot`). The CDK automatically creates an `osClusterAccessSG` security group, which is applied to the migration services. The user should then add this security group to their existing domain to allow access.
```

File: _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md [hunk 18]
```diff
@@ -7,25 +7,27 @@ parent: Deploying migration assistant
...
-### OpenSearch Service
+Use the following scenarios for OpenSearch Service or Serverless target clusters:
+
+### OpenSearch service
 
 For an OpenSearch Domain, two main configurations are typically required to ensure proper functioning of the migration solution:
 
-1. **Security Group Configuration**:  
-   The Domain should have a security group that allows communication from the applicable Migration services (Traffic Replayer, Migration Console, Reindex-from-Snapshot). The CDK will automatically create an `osClusterAccessSG` security group, which is applied to the Migration services. The user should then add this security group to their existing Domain to allow access.
+1. **Security Group Configuration**
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md:20]: Should this be in sentence case (fine as is if it's a UI element that's capitalized)?

File: _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md [hunk 19]
```diff
@@ -7,25 +7,27 @@ parent: Deploying migration assistant
...
+
+   The Domain should have a security group that allows communication from the applicable Migration services (Traffic Replayer, Migration Console, `Reindex-from-Snapshot`). The CDK automatically creates an `osClusterAccessSG` security group, which is applied to the migration services. The user should then add this security group to their existing Domain to allow access.
 
-2. **Access Policy Configuration**:  
-   The Domain’s access policy should either:
-   - Be an open access policy that allows all access, or
-   - Be configured to allow at least the IAM task roles for the applicable Migration services (Traffic Replayer, Migration Console, Reindex-from-Snapshot) to access the Domain.
+2. **Access Policy Configuration** should be one of the following:
+   - An open access policy that allows all access.
+   - Configured to allow at least the IAM task roles for the applicable Migration services (Traffic Replayer, Migration Console, `Reindex-from-Snapshot`) to access the Domain.
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md:26]:
```suggestion
   - Configured to allow at least the AWS Identity and Access Management (IAM) task roles for the applicable migration services (Traffic Replayer, Migration Console, `Reindex-from-Snapshot`) to access the domain.
```

File: _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md [hunk 20]
```diff
@@ -7,25 +7,27 @@ parent: Deploying migration assistant
...
+Use the following scenarios for OpenSearch Service or Serverless target clusters:
+
+### OpenSearch service
 
 For an OpenSearch Domain, two main configurations are typically required to ensure proper functioning of the migration solution:
 
-1. **Security Group Configuration**:  
-   The Domain should have a security group that allows communication from the applicable Migration services (Traffic Replayer, Migration Console, Reindex-from-Snapshot). The CDK will automatically create an `osClusterAccessSG` security group, which is applied to the Migration services. The user should then add this security group to their existing Domain to allow access.
+1. **Security Group Configuration**
+
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md:21]: To the best of my knowledge, "domain" is not capitalized as a proper noun. Please lowercase across this and the other files.

File: _migrations/deploying-migration-assistant/index.md [hunk 21]
```diff
@@ -1,5 +1,9 @@
...
-title: Deploying migration assistant
+title: Deploying Migration Assistant
 nav_order: 10
----
\ No newline at end of file
+---
+
+# Deploying Migration Assistant
+
+This section details what options are available for deploying Migration Assistant.
```
Comment (natebower) [inline _migrations/deploying-migration-assistant/index.md:9]:
```suggestion
This section provides information about the available options for deploying Migration Assistant.
```

**Review by natebower — APPROVED** [review]
@Naarcha-AWS Please see my comments and changes and let me know if you have any questions. Thanks!

File: _migrations/deploying-migration-assistant/configuration-options.md [hunk 22]
```diff
@@ -7,66 +7,35 @@ parent: Deploying migration assistant
...
-This page outlines the configuration options for three key migrations:
-1. **Metadata Migration**
-2. **Backfill Migration with Reindex-from-Snapshot (RFS)**
-3. **Live Capture Migration with Capture and Replay (C&R)**
+This page outlines the configuration options for three key migrations scenarios:
 
-Each of these migrations may depend on either a snapshot or a capture proxy. The CDK context blocks below are shown as separate context blocks for each migration type for simplicity. If performing multiple migration types, combine these options, as the actual execution of each migration is controlled from the Migration Console.
+1. **Metadata migration**
+2. **Backfill migration with `Reindex-from-Snapshot` (RFS)**
+3. **Live capture migration with Capture and Replay (C&R)**
```
[the remark this answers is not in the document]
Comment (Naarcha-AWS) [reply _migrations/deploying-migration-assistant/configuration-options.md:14]: Switched to capitalized.

File: _migrations/deploying-migration-assistant/configuration-options.md (hunk 12 above)
Comment (Naarcha-AWS) [inline _migrations/deploying-migration-assistant/configuration-options.md:124]:
```suggestion
Both the source and target cluster can use no authentication, authentication limited to VPC, basic authentication with a username and password, or AWS Signature Version 4 scoped to a user or role.
```

File: _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md (hunk 18 above)
Comment (Naarcha-AWS) [reply _migrations/deploying-migration-assistant/iam-and-security-groups-for-existing-clusters.md:20]: These are UI elements.

**Pull request merged**